# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/TestMP3Demuxer.cpp
# Commit: de21e15f438f
# Full Hash: de21e15f438fd87cf8b7f41dc98997fcf0f8c821
# Author: alwu <alwu@mozilla.com>
# Date: 2023-08-01 16:06:52
# Regressor Bug: 1845213
# File Overlap Count: 1
# Description:
#   Bug 1845213 - part2 : round up the inbase to make the result more precise. r=padenot
#   
#   Eg. {448/48,000} - {1/1,000,000} without rounding, the result would be
#   {447/48,000}, which loses around 20ms precision.
#   
# ==============================================================================

diff -r 3241b87d217e -r de21e15f438f dom/media/gtest/TestMP3Demuxer.cpp
--- a/dom/media/gtest/TestMP3Demuxer.cpp	Tue Aug 01 00:02:39 2023 +0000
+++ b/dom/media/gtest/TestMP3Demuxer.cpp	Tue Aug 01 00:02:39 2023 +0000
@@ -106,7 +106,7 @@
       // The tolerance comes from the fact that this file has ID3v1 information
       // at the end, this trips our CBR duration calculation. The file has
       // however the correct duration when decoded / demuxed completely.
-      res.mDuration = Some(MP3Resource::Duration{30093062, 0.00015f});
+      res.mDuration = Some(MP3Resource::Duration{30093063, 0.00015f});
       res.mSeekError = 0.02f;
       res.mSampleRate = 44100;
       res.mSamplesPerFrame = 1152;